### PR TITLE
Add system commands

### PR DESCRIPTION
**Pull request by dpgraham**

### CODE REVIEW

File: commands-yml/commands/device/system/system-time.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    uiautomation: true
+  android:
+    uiautomator2: true
+    uiautomator: true
+  mac:
+    mac: false
+  windows:
+    windows: false
+client_support:
+  java: false
```
Comment (vmaxim) [inline commands-yml/commands/device/system/system-time.yml]: should be true for java and python

File: commands-yml/commands/device/system/system-time.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  python: false
+  ruby: true
+  php: true
+  csharp: true
+  javascript_wd: true
+  javascript_wdio: true
+
+# Information about the HTTP endpoints
+endpoint:
+  url: /session/:session_id/device/system_bars
```
Comment (vmaxim) [inline commands-yml/commands/device/system/system-time.yml]: should be /session/:sessionId/device/system_time with GET method

File: commands-yml/commands/device/network/gsm-voice.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  python: false
+  ruby: true
+  php: true
+  csharp: true
+  javascript_wd: true
+  javascript_wdio: true
+
+# Information about the HTTP endpoints
+endpoint:
+  url: /session/:session_id/device/gsm_signal
```
Comment (vmaxim) [inline commands-yml/commands/device/network/gsm-voice.yml]: /session/:session_id/device/gsm_voice

File: commands-yml/commands/device/network/toggle-location-services.yml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    uiautomation: false
+  android:
+    uiautomator2: true
+    uiautomator: true
+  mac:
+    mac: false
+  windows:
+    windows: false
+client_support:
+  java: false
```
Comment (vmaxim) [inline commands-yml/commands/device/network/toggle-location-services.yml]: Should be `true`.

File: commands-yml/commands/device/network/toggle-location-services.yml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+endpoint:
+  url: /session/:session_id/device/toggle_location_services
+  method: POST
+  url_parameters:
+    - name: session_id
+      description: ID of the session to route the command to
+
+# Links to specifications. Should link to at least one specification
+specifications: 
+  jsonwp: https://github.com/appium/appium-base-driver/blob/master/lib/mjsonwp/routes.js#L402
```
Comment (vmaxim) [inline commands-yml/commands/device/network/toggle-location-services.yml]: routes.js#405

File: commands-yml/commands/device/system/open-notifications.yml [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+    uiautomation: false
+  android:
+    uiautomator2: true
+    uiautomator: true
+  mac:
+    mac: false
+  windows:
+    windows: false
+client_support:
+  java: false
```
Comment (vmaxim) [inline commands-yml/commands/device/system/open-notifications.yml]: Should be `true`.

**Review by SrinivasanTarget — CHANGES_REQUESTED** [review]
GSM endpoints doesn't work in UIAutomator2 endpoints. Ping @vrunoa @dpgraham . It works with UIAutomator1 mode still.

File: commands-yml/commands/device/keys/hide-keyboard.yml [hunk 7]
```diff
@@ -0,0 +1,82 @@
+---
+name: Find Element
```
Comment (imurchie) [inline commands-yml/commands/device/keys/hide-keyboard.yml]: None of this corresponds to `hideKeyboard`.

File: commands-yml/commands/device/network/gsm-call.yml [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+  php: "https://github.com/appium/php-client/" # TODO
+  csharp: "https://github.com/appium/appium-dotnet-driver/" # TODO
+
+# Driver support by platform
+driver_support:
+  ios:
+    xcuitest: false
+    uiautomation: false
+  android:
+    uiautomator2: false
```
Comment (imurchie) [inline commands-yml/commands/device/network/gsm-call.yml]: Does this not just fall through to the android driver?

File: docs/toc.js [hunk 9]
```diff
@@ -46,7 +46,28 @@ module.exports = {
           ["Active Element", "active.md"],
           ["Equals Element", "equals-element.md"],
         ]]
-      ]]
+      ]],
+      ["Device", ["device",
+        ["Keys", ["keys",
+          ["Hide Keyboard", "hide-keyboard.md"],
+          ["Is Keyboard Shown", "is-keyboard-shown.md"],
```
Comment (imurchie) [inline docs/toc.js]: Is this file in some other PR?

**Review by imurchie — CHANGES_REQUESTED** [review]

File: commands-yml/commands/device/network/gsm-call.yml (hunk 8 above)
Comment (dpgraham) [reply commands-yml/commands/device/network/gsm-call.yml]: I don't know. @SrinivasanTarget you were saying this doesn't work on UiAutomator2?

Comment (SrinivasanTarget) [reply commands-yml/commands/device/network/gsm-call.yml]: Yes it wont work in UIAutomator2 mode presently. Have already tested it and updated @vrunoa on this.

File: docs/toc.js (hunk 9 above)
Comment (dpgraham) [reply docs/toc.js]: Added missing file

**Pull request merged**